sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/init.sql

```diff
@@ -0,0 +1,19 @@
+-- Generated by ./bqetl generate events_daily
+CREATE OR REPLACE TABLE
+  fenix_derived.google_ads_campaign_cost_breakdowns_v1
+PARTITION BY
+  date
+OPTIONS
+  (require_partition_filter = TRUE)
+AS
+SELECT
+  CAST(NULL AS date) AS date,
+  CAST(NULL AS STRING) AS campaign_name,
+  CAST(NULL AS FLOAT64) AS campaign_spend_in_cents,
+  CAST(NULL AS INT64) AS ad_impressions,
+  CAST(NULL AS INT64) AS installs,
+  CAST(NULL AS INT64) AS dous,
+  CAST(NULL AS INT64) AS ad_clicks,
+  CAST(NULL AS FLOAT64) AS cost_per_install_microcents,
```

Comment:
I put spend in cents, but left the calculations BASED on spend in mircocents, because we're dividing by very large numbers in many cases and the cost of things in cents would be .0000000somethingsomething.

Reply:
Nice, I agree with this approach 👍

Reply:
Are you going to add a view that converts this to $$?

Reply:
Well..if I convert it to dollars it's gonna be _really_ small numbers. Unless we make a second, aggregate table that reconstitutes these numbers as "spend per campaign total" or something. That...I think is pretty crept scope from the ticket, but if that's what they need then that's what they need and we can ticket/execute for that.

Reply:
It looks like this _is_ dollars, not cents: https://fivetran.com/docs/applications/google-ads

Reply:
<img width="623" alt="Screen Shot 2022-12-21 at 12 56 31 PM" src="https://user-images.githubusercontent.com/20819040/208972482-18490585-5c50-45fc-847a-a7a69f2b82fa.png">

Reply:
Eeeeeenteresting. All right, I'll go update that!